projects/core/src/occ/adapters/checkout/default-occ-checkout-config.ts

```diff
@@ -6,7 +6,7 @@ export const defaultOccCheckoutConfig: OccConfig = {
       endpoints: {
         setDeliveryAddress:
           'users/${userId}/carts/${cartId}/addresses/delivery',
-        placeOrder: 'users/${userId}/orders',
+        placeOrder: 'users/${userId}/orders?fields=FULL',
```

Comment:
Can you be more specific?

Reply:
answered below, but it was already there and I don't know which exact properties it needs but the properties are the ones we need to populate the items. I believe it's best to keep it as how it was done before